packages/flutter_tools/lib/src/build_system/targets/macos.dart

```diff
@@ -83,111 +125,195 @@ class UnpackMacOS extends Target {
   }
 }
 
-/// Tell cocoapods to re-fetch dependencies.
-class DebugMacOSPodInstall extends Target {
-  const DebugMacOSPodInstall();
+/// Compile an App.framework for a macOS target device.
+class MacOSAotAssembly extends Target {
+  const MacOSAotAssembly();
 
   @override
-  String get name => 'debug_macos_pod_install';
+  String get name => 'macos_aot_assembly';
 
   @override
-  List<Source> get inputs => const <Source>[
-    Source.artifact(Artifact.flutterMacOSPodspec,
-      platform: TargetPlatform.darwin_x64,
-      mode: BuildMode.debug
-    ),
-    Source.pattern('{PROJECT_DIR}/macos/Podfile', optional: true),
-    Source.pattern('{PROJECT_DIR}/macos/Runner.xcodeproj/project.pbxproj'),
-    Source.pattern('{PROJECT_DIR}/macos/Flutter/ephemeral/Flutter-Generated.xcconfig'),
-  ];
+  Future<void> build(List<File> inputFiles, Environment environment) async {
+    final AOTSnapshotter snapshotter = AOTSnapshotter(reportTimings: false);
```

Comment:
The PR comment suggests that AOT isn't actually wired up yet; why is AOT code being added?

Reply:
I'm refactoring all of the real AOT code to a second PR